alternator/executor.cc

```diff
@@ -2903,16 +2934,12 @@ future<executor::request_return_type> executor::batch_write_item(client_state& c
             "maximum is {} (from configuration variable alternator_max_items_in_batch_write)", total_items, maximum_batch_write_size));
     }
     bool should_add_wcu = wcu_consumed_capacity_counter::should_add_capacity(request);
-    size_t wcu_put_units;
-    size_t wcu_delete_units;
     rjson::value consumed_capacity = rjson::empty_array();
     std::vector<std::pair<schema_ptr, put_or_delete_item>> mutation_builders;
-    std::vector<std::tuple<lw_shared_ptr<stats>, size_t, size_t>> per_table_wcu;
+    std::vector<std::tuple<lw_shared_ptr<stats>, schema_ptr, std::string>> per_table_wcu;
```

Comment:
I don't know what this change is. This is a big patch doing a lot of different things, and I think you described all of them in the commit message, but it's still hard for me to understand what each piece does. I think it will be helpful if you split this patch.